public/src/inverted-index.js

```diff
@@ -14,49 +15,52 @@ class InvertedIndex {
     this.search_terms = [];
   }
   /**
+   * Create `index
+   *
    * creates an inverted index from the specified fileName or JsonObject
-   * @method createIndex
-   * @param {String} fileName
-   * @param {Object} jsonObject
+   * @param {Object} jsonObject the json data to build
    * @return {Object} index that was built
    */
-  createIndex(fileName, jsonObject) {
+  createIndex(jsonObject) {
```

Comment:
How about using better describing variable names (avoid including data types in your variable names)